### ISSUE TITLE
Segfault when attempting to compile Pyre with dune 1.8.0

### ISSUE THREAD

**Issue by sinancepel** [ocaml/dune#1918]
We've been using Dune to build Pyre (https://github.com/facebook/pyre-check) for a while now. After switching to dune 1.8.0 from dune 1.2.1, I'm seeing crashes of the form `Error: exception Invalid_argument("atom '' cannot be in dune syntax")`.

Any pointers on how I could debug this and support later Dune versions? I tried running it with `dune build @install --verbose` and with the various debug flags, but none seemed to point me at the right file with the syntax error.

To repro: 

```
$ dune --version
1.8.0
$ git clone git@github.com:facebook/pyre-check.git
$ cd pyre-check
$ ./scripts/setup.sh
```

**Comment by rgrinberg** [comment]
I fixed it in #1919. It seems like your version is empty by the way.

**Comment by rgrinberg** [comment]
Fixed in 1.8.1

**Comment by sinancepel** [comment]
Amazing, thank you!
